tests/test_migrate.py

```diff
@@ -104,49 +104,57 @@ def test_migrate_transform_job_id(migrator: Migrator):
     assert migrate_batch.transform_job_id == "xxx"
 
 
-def test_migrate_batch_records_success(migrator: Migrator, md_fixtures: list[Fixture]):
-    batch, _ = get_transform_results("noop", migrator)
-    migrate_batch = migrator.migrate(batch, overwrite=False, group=None)
+def test_migrate_batch_records_success(
+    migrator: Migrator, md_fixtures: list[Fixture], group_fixture: int
+):
+    batch, _ = get_transform_results("change-language", migrator)
+    migrate_batch = migrator.migrate(batch, overwrite=False, group=group_fixture)
+    assert len(migrate_batch.successes())
     for record in migrate_batch.successes():
         assert record.source_uuid in [f.uuid for f in md_fixtures]
         assert record.target_uuid not in [f.uuid for f in md_fixtures]
         assert record.url == GN_TEST_URL
-        assert record.source_content is not None  # TODO: check content when formatting is done
-        assert record.target_content is not None  # TODO: check content when formatting is done
+        assert record.source_content is not None
+        assert record.target_content is not None
+        assert record.source_content != record.target_content
         assert record.template is False
 
 
 def test_migrate_batch_records_failure(migrator: Migrator, md_fixtures: list[Fixture]):
-    batch, _ = get_transform_results("noop", migrator)
+    batch, _ = get_transform_results("change-language", migrator)
     with patch("ecospheres_migrator.geonetwork.GeonetworkClient.update_record") as mocked_method:
         mocked_method.side_effect = Exception("Mocked update_record error")
         migrate_batch = migrator.migrate(batch, overwrite=False, group=None)
+    assert len(migrate_batch.failures())
     for record in migrate_batch.failures():
         assert record.source_uuid in [f.uuid for f in md_fixtures]
         assert record.url == GN_TEST_URL
-        assert record.source_content is not None  # TODO: check content when formatting is done
-        assert record.target_content is not None  # TODO: check content when formatting is done
+        assert record.source_content is not None
+        assert record.target_content is not None
+        assert record.source_content != record.target_content
         assert record.template is False
         assert record.error is not None  # actual error is tested below
 
 
 def test_migrate_overwrite_gn_error(migrator: Migrator, md_fixtures: list[Fixture]):
-    batch, _ = get_transform_results("noop", migrator)
+    batch, _ = get_transform_results("change-language", migrator)
     with patch("ecospheres_migrator.geonetwork.GeonetworkClient.update_record") as mocked_method:
         mocked_method.side_effect = Exception("Mocked update_record error")
         migrate_batch = migrator.migrate(batch, overwrite=True, group=None)
     assert len(migrate_batch.failures()) == len(md_fixtures)
     assert migrate_batch.mode == MigrateMode.OVERWRITE
+    assert len(migrate_batch.failures())
```

Comment:
Already tested 2 lines above?

Reply:
True! I wanted to make sure I failsafed all my loopy tests, guess I made extra sure.